packages/grid/x-data-grid-premium/package.json

```diff
@@ -51,7 +51,7 @@
     "@mui/x-license-pro": "7.0.0-alpha.8",
     "@types/format-util": "^1.0.4",
     "clsx": "^2.1.0",
-    "exceljs": "^4.3.0",
+    "exceljs": "^4.4.0",
```

Comment:
Do we really need to bump the version here?
@alexfauquette

Reply:
I don't think it's necessary. We could update the docs and let package dependency untouched

Reply:
Need - probably not, unless there are some bugfixes we want to benefit from.
But this dependency is not a `peer` one - we are responsible for bumping it.
Correct me if I'm wrong, but if people are not using it in their own codebase and not refreshing a lockfile - they will always be stuck with the version that got installed at the time the dep was added. 🤔

Reply:
From what I understand (I'm not fully aware of dependencies management)

If we have `"exceljs": "^4.3.0",` and a user set `"exceljs": "^4.4.0",` in his codebase, it wil reolve with something like

```
"exceljs@^4.3.0", "exceljs@^4.4.0":
  version "4.4.0"
  resolved: ....
```
because the v4.4.0 resolve both ^4.3.0 and ^4.4.0

So we let the freedom of users to keep the 4.3.0

Reply:
Well. It's your decision.
IMHO, that's perfectly valid approach in case of `peer` dependencies.
But given that this is a direct dependency, I'd be in favor of bumping it in the package itself.

I'm leaving this topic in the hands of @mui/xgrid and/or @alexfauquette. 👌

Reply:
> if people are not using it in their own codebase and not refreshing a lockfile - they will always be stuck with the version that got installed at the time the dep was added. 🤔

Good point. Unless they intentionally do `npm update`, the dependency won't update automatically.

`exceljs` dependency is special - initially, it was a peer dependency, but then we made it a dependency to solve some issues: https://github.com/mui/mui-x/pull/4939

I'm OK with bumping the version in the data grid package 👍🏻

Reply:
Hm, it's good that you linked that PR.
It raises some questions:
1. Was the move to a direct dependency warranted?
2. If it was only because of codesandbox, we can update the dependencies resolution to include `exceljs` in cases where it's needed, just how [we do](https://github.com/mui/mui-x/blob/next/docs/src/modules/utils/postProcessImport.ts) with various date adapters.

Reply:
It wasn't because of the Codesandbox - see https://github.com/mui/material-ui/pull/32824/files#r876884180
We couldn't figure out how to make it optional, so we moved it to dependencies to make sure it's always installed.

Reply:
I saw https://github.com/mui/material-ui/pull/32824/files#r876982792 and thought that you had a possible workaround. 🤔 
But that would probably hardly help with codesandbox issue. 🙈

Reply:
I didn't like that workaround 😅